src/main/scripts/janusgraph-python

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+
+from gremlin_python.process.graph_traversal import __
+from janusgraph_python.core.attribute.TextPredicate.Text import Text
+from janusgraph_python.core.datatypes.GeoShape import GeoShape
+from janusgraph_python.core.attribute import Geo
+
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+import pprint
+
+
+if __name__ == '__main__':
+
+    connection = JanusGraphClient().connect(url="10.153.22.172", port="8182", graph="gg").get_connection()
```

Comment:
`0.0.0.0`?

Reply:
Again, this file won't be present in recent commit.

Reply:
@sjudeng : Added copyright to this file as of now. Don't know why I'm unable to add this file to .gitignore. somehow I've added the file but it still shows up on my Git, and unable to remove from repo anymore without deleting the file actually.
Anyways, once we move towards 1.0.0 release, I'll just delete them from codebase.